### ISSUE TITLE
Specify: digital material with no relation to other entities

### ISSUE THREAD

**Issue by MortenHofft** [gbif/model-material#68]
line 48815 in ENTITY.csv
`548631fb-a0b5-11ed-803c-6c2b597b15e2,DIGITAL_ENTITY,https://www.gbif.org/dataset/8f79c802-a58c-447f-99aa-1d6a0790825a,,`

That ID only appears ones in the entity_relationships file
`e65f79ba-0c0c-ee21-434e-ad0d558c3b7e,,548631fb-a0b5-11ed-803c-6c2b597b15e2,has genetic sequence,,https://www.ncbi.nlm.nih.gov/nuccore/AB889686,,0`

How am I to interpret that? Shouldn't it be related to some collection object (`entityType=MATERIAL_ENTITY`)?

It seems to be the case for all entries in the relation table with type `has genetic sequence`

**Comment by acbentley** [comment]
As far as I know, no.  I do not have material_entities for these.  All I have is the sequence.  This goes to our conversation earlier about the difference between a genetic sample and a genetic sequence and why we deleted the genetic sequence table.  It also goes to our conversation about incorporating my tissue collection ad these all relate to tissue samples in that collection.

**Comment by MortenHofft** [comment]
ohh - so for now there are no analog entities. Only a database of ~sequences~ links to where sequences are (e.g. https://www.ncbi.nlm.nih.gov/nuccore/AB889686).

Does it still make sense to publish? Right now there are no data other than a link - no relation to other entities. No determination. No meta data at all. Or am I failing to unfold it?

**Comment by tucotuco** [comment]
It still makes sense to publish them, but as described above.

**Comment by MortenHofft** [comment]
Thanks @tucotuco that would make sense to me and what I would expect given that the tissue isn't there. The genetic relation would then "snap" to the parent material to not be free floating links.

**Comment by tucotuco** [comment]
Exactly.

**Comment by acbentley** [comment]
I used the wrong GUID field to connect.  Repaired in the below.  I have also switched the subject_id and object_id and entity_relationship type for images as per #67.  I have done this manually in the export file so as not to have to go through the concatenation process again.  The original queries now also reflect the change by switching the fields.

[ENTITY_RELATIONSHIP.csv](https://github.com/gbif/model-material/files/10735058/ENTITY_RELATIONSHIP.csv)
